webpack/reducers/reducer_generator.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+
```

Comment:
Yes. The reducer generator allows us to get rid of a bunch of reducers that were 90% the same.